packages/workflow/src/workflow.ts

```diff
@@ -868,6 +870,19 @@ export function workflowInfo(): WorkflowInfo {
   return activator.info;
 }
 
+/**
+ * Get information about the current update if any.
+ *
+ * @return Info for the current update handler the code calling this is executing
+ * within if any.
+ *
+ * @experimental
+ */
+export function currentUpdateInfo(): UpdateInfo | undefined {
```

Comment:
I'd call it `updateInfo()`, similar to `workflowInfo()` and `activityInfo()`.

Reply:
I'm following the convention discussed in https://github.com/temporalio/sdk-python/pull/544 where @cretz suggested to use `current` because it is only available within the scope of a handler execution. But I'm happy to change that, if it is ok to be different across languages...

Reply:
Arguable, but fine... Let's align with python.